From 6856321053ed8dfd8991e3a8eabba8e483967138 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=E9=BB=84=E6=B6=9B?= <huangtao@rock-chips.com>
Date: Sat, 30 Jul 2011 22:32:35 +0800
Subject: [PATCH] Revert "Bluetooth: Fallback to SCO on error code 0x10
 (Connection Accept Timeout)."

This reverts commit df28bc9c1957170dbf796063853a4a1aee1af6ac.
---
 net/bluetooth/hci_event.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/net/bluetooth/hci_event.c b/net/bluetooth/hci_event.c
index 10edd1a71f08..5c495893bc07 100644
--- a/net/bluetooth/hci_event.c
+++ b/net/bluetooth/hci_event.c
@@ -1698,7 +1698,6 @@ static inline void hci_sync_conn_complete_evt(struct hci_dev *hdev, struct sk_bu
 		hci_conn_add_sysfs(conn);
 		break;
 
-	case 0x10:	/* Connection Accept Timeout */
 	case 0x1c:	/* SCO interval rejected */
 	case 0x1a:	/* unsupported feature */
 	case 0x1f:	/* Unspecified error */
-- 
2.35.3

